### PR TITLE
TA#26976 Fix lang filter

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

File: src/core.py [hunk 1]
```diff
@@ -132,20 +131,15 @@ def _move_modules(self, temp_repo, destination):
...
         :param string destination: the folder where the add-on should end up at.
         """
-        folders = self._get_module_folders(temp_repo)
-        for folder in folders:
+        for folder in self._iter_module_folders(temp_repo):
             force_move(folder, destination)
 
-    def _get_module_folders(self, temp_repo):
-        """Get a list of module paths contained in a temp directory.
+    def _iter_module_folders(self, temp_repo):
```
Comment (ddufresne) [inline src/core.py:137]: Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/Numigi/gitoo/pullRequest?prid=6195482)

Comment (foutoucour) [inline src/core.py:137]: Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/numigi/gitoo/pullRequest?prid=6195483)

File: src/core.py [hunk 2]
```diff
@@ -132,20 +131,15 @@ def _move_modules(self, temp_repo, destination):
...
+        """Iter over a list of module paths contained in the git directory.
```
Comment (ddufresne) [inline src/core.py:138]: Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/Numigi/gitoo/pullRequest?prid=6195482)

File: src/core.py [hunk 3]
```diff
@@ -132,33 +131,17 @@ def _move_modules(self, temp_repo, destination):
...
-
-        :param string module: the name of the file/folder
-        :rtype: bool
-        """
-        file_path = os.path.join(repo_path, file_name)
-        return (
-            os.path.isdir(file_path) and
-            '__manifest__.py' in os.listdir(file_path)
-        )
+    def _iter_modules(self, temp_repo):
```
Comment (foutoucour) [inline src/core.py:143]: Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/numigi/gitoo/pullRequest?prid=6195483)

Comment (ddufresne) [inline src/core.py:143]: Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/Numigi/gitoo/pullRequest?prid=6195482)

File: src/core.py [hunk 4]
```diff
@@ -132,33 +131,18 @@ def _move_modules(self, temp_repo, destination):
...
-
-        :param string temp_repo: the folder containing the modules.
-        """
-        paths = (
-            os.path.join(temp_repo, path) for path in os.listdir(temp_repo)
-            if self._is_odoo_module(temp_repo, path) and self._is_module_included(path)
-        )
-        return (path for path in paths if os.path.isdir(path))
+    def _iter_included_modules(self, temp_repo):
+        for path in self._iter_modules(temp_repo):
```
Comment (ddufresne) [inline src/core.py:138]: Codacy found an issue: [No value for argument 'temp_repo' in staticmethod call](https://app.codacy.com/gh/Numigi/gitoo/pullRequest?prid=6195482)

Comment (foutoucour) [inline src/core.py:138]: Codacy found an issue: [No value for argument 'temp_repo' in staticmethod call](https://app.codacy.com/manual/numigi/gitoo/pullRequest?prid=6195483)

**Pull request merged**